src/main/java/Ui.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+/**
+ * Deals with interactions with the user
+ */
+public class Ui {
+    Ui() {
+    }
+
+    /**
+     * Prints the welcome statement.
+     */
+    public void start() {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Prints the goodbye statement.
+     */
+    public void finalBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Prints the error message for loading error.
+     */
+    public void showLoadingError() {
+        System.out.println("System data could not be loaded!");
+    }
+
+    /**
+     * Prints the error message for empty description.
+     */
+    public void emptyInput() {
+        System.out.println(" ☹ OOPS!!! The description of a todo cannot be empty.");
+    }
+
+    /**
+     * Prints the error message for invalid commands.
+     */
+    public void doNotUnderstand() {
+        System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+    }
```

Comment:
Should this method name be a verb instead?
```suggestion
    public void displayDoNotUnderstandError() {
        System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
    }
```